### PR TITLE
Add validation component

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. Accepts a list of messages
1. Accepts a boolean for triggered or passive state
1. Can wrap any element
1. Accepts argument to render messages before or after children
1. Ability to render messages to other components

### CODE REVIEW

**Comment by natestormer** [comment]
:m:

**Comment by natestormer** [comment]
:star2:

File: components/form/validation/validation.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    {
+        return {
+            componentCSSClassName        : 'validation',
+            show                         : false,
+            renderMessages               : 'after',
+            messageContainer             : null,
+            messageContainerProps        : null,
+            messages                     : null
+        };
+    },
```
Comment (kylecesmat) [inline components/form/validation/validation.jsx]: spacing

File: components/form/validation/validation.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+    mixins : [SynfrastructureHelperMixin],
+
+    propTypes : {
+        componentCSSClassName        : React.PropTypes.string,
+        show                         : React.PropTypes.bool,
+        renderMessages               : React.PropTypes.oneOf(['before', 'after']),
+        messageContainer             : React.PropTypes.func,
+        messageContainerProps        : React.PropTypes.object,
+        messages                     : React.PropTypes.arrayOf(
```
Comment (kylecesmat) [inline components/form/validation/validation.jsx]: spacing

**Comment by kylecesmat** [comment]
:interrobang: just a few things to fix and then you can +1 it

**Comment by natestormer** [comment]
:star2: @kylecesmat

**Comment by natestormer** [comment]
:+1:

**Pull request merged**